installing/installing_with_agent_based_installer/installing-using-iscsi.adoc

```diff
@@ -0,0 +1,59 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="installing-using-iscsi"]
+= Preparing installation assets for iSCSI booting
+include::_attributes/common-attributes.adoc[]
+:context: installing-using-iscsi
+
+toc::[]
+
+You can boot an {product-title} cluster through iSCSI using an ISO image generated by the Agent-based Installer.
+The following procedures describe how to prepare the necessary installation resources for you to upload to an iSCSI target.
+
+The assets you create in these procedures will deploy a single-node {product-title} installation.
+You can use these procedures as a basis and modify configurations according to your requirements.
+
+[id="iscsi-boot-requirements_{context}"]
+== Requirements for iSCSI booting
+
+The following configurations are necessary to enable iSCSI booting using the Agent-based Installer:
+
+* Dynamic Host Configuration Protocol (DHCP) must be configured.
+Static networking is not supported.
+* You must create an additional network for iSCSI that is separate from the machine network of the cluster.
+The machine network is rebooted during cluster installation and cannot be used for the iSCSI session.
+* If the iSCSI server uses disks, the iSCSI disk must be specified using the `rootDeviceHints` parameter.
```

Comment:
This should refer to the actual host you are installing the agent ISO on, not the ISCSI server.
Perhaps - "ff the host on which you are booting the agent ISO also has an installed disk, it may be necessary to specify the iSCSI disk name in the `rootDeviceHints` parameter to ensure that it is picked as the boot disk for the final RHCOS image."